sql/service/src/main/java/org/apache/spark/sql/service/cli/CLIService.java

```diff
@@ -209,8 +209,21 @@ public void closeSession(SessionHandle sessionHandle)
   @Override
   public GetInfoValue getInfo(SessionHandle sessionHandle, GetInfoType getInfoType)
       throws ServiceSQLException {
-    GetInfoValue infoValue = sessionManager.getSession(sessionHandle)
-        .getInfo(getInfoType);
+    GetInfoValue infoValue = null;
+    switch (getInfoType) {
+      case CLI_SERVER_NAME:
+        infoValue = new GetInfoValue("Spark SQL");
```

Comment:
BTW: I recently learned that some drivers, e.g. Simba drivers do depend on this being "Spark SQL" for some internal version checks they make, so good not to change it :+1: